libbeat/docs/monitoring/monitoring-beats.asciidoc

```diff
@@ -40,7 +40,7 @@ the same {es} cluster, specify the following minimal configuration:
 --
 ["source","yml",subs="attributes"]
 --------------------
-xpack.monitoring:
+monitoring:
```

Comment:
Should this be just be `monitoring`?

Reply:
yes, for 7.2 and later. If you decide to backport to 7.1 or earlier, you'll need to change the name as needed.